### DISCUSSION TITLE
Interpretation of output spanning reads and discordant reads

### DISCUSSION THREAD

**Discussion by ashleyzhou972** [stjude/CICERO#63] [Q&A] [answered]
I was wondering if there's a more detailed description of the output columns,
The only resource I could find was here: https://university.stjude.cloud/docs/genomics-platform/workflow-guides/rapid-rnaseq/#interpreting-results, and the [FusionEditor help page](https://docs.google.com/document/d/1LgcMk_p1qyPgFPQChy4sLlU9GDL6oahffP7jJfD8ZwE/edit). Neither of those resources contain definitions for columns such as `matchA`, `repeatA`, `coverageA`, or `ratioA`.

Specifically, I have the following questions:
- If the `readsA` and `readsB` columns in `final_fusions.txt` are columns containing number of **chimeric reads** for a fusion, why are the two having different values for the same fusion? By definition a chimeric read should contain sequences from both A and B genes.

- Are there metrics equivalent to star-fusion's `JunctionReadCount` and `SpanningFragCount` columns? According to STAR-Fusion [wiki](https://github.com/STAR-Fusion/STAR-Fusion/wiki#output-from-star-fusion), the former  "indicates the number of RNA-Seq fragments containing a read that aligns as a split read at the site of the putative fusion junction", and the latter "indicates the number of RNA-Seq fragments that encompass the fusion junction such that one read of the pair aligns to a different gene than the other paired-end read of that fragment".

If you could illustrate how CICERO's output `readsA` and `readsB` columns (or maybe other columns) relate to these two concepts, to enable us to do a more thorough comparison of the two methods, that would be great!

Thank you!

**Comment by adthrasher** [comment]
The FusionEditor has a [link](https://docs.google.com/document/d/1jkVYRPIJpkWvA9vqtahRlNn63Hk5DehjHbF_BH9k7Rs/edit) to the file format. @liqingti - Can you answer the questions about the read counts?

**Reply by adthrasher** [reply]
I believe that readsA indicates reads that were originally aligned at posA. So it is based on the original number of aligned reads at each breakpoint.

**Reply by ahdee** [reply]
@adthrasher were you able to get any more info? I second this question.  It would be valuable to have both the junction and discordant reads, the reason is because it allows a way to calculate relative abudance when normalized to the library.

**Comment by liqingti** [comment, marked as answer]
Q1: readsA and readsB stands for "number of junction reads that support the fusion at breakpoint A / B".
For most cases, it equals the number of supporting soft-clip reads at each breakpoint.

Q2: basically JunctionReadCount in STAR-Fusion approximately equals readsA+readsB in CICERO. We do not have similar output of "SpanningFragCount" in CICERO.
